### PR TITLE
Noita: Support for the new biome

### PR DESCRIPTION
**Pull request by ScipioWright**
## What is this fixing or adding?
Added support for the new biome added to the Noita beta branch a few hours ago.

## How was this tested?
Successfully generated the world, and successfully got checks to spawn in the world.

## If this makes graphical changes, please attach screenshots.
meat realm

### CODE REVIEW

**Comment by ThePhar** [comment]
I'm assuming #2040 is a duplicate then?

**Comment by ScipioWright** [comment]
Yup, sorry about that -- I pulled it then noticed it pulled a lot of duplicate commits with it. Figured it was easier at that point to murder it and rewrite the update since it is a small one.

File: worlds/noita/Options.py [hunk 1]
```diff
@@ -7,11 +7,13 @@ class PathOption(Choice):
     Main Path includes the main 7 biomes you typically go through to get to the final boss.
     Side Path includes the Lukki Lair and Fungal Caverns. 9 biomes total.
     Main World includes the full world (excluding parallel worlds). 14 biomes total.
+    Beta Biome includes the Main World as well as the new beta biome. Note that it is fairly small.
     Note: The Collapsed Mines have been combined into the Mines as the biome is tiny."""
     display_name = "Path Option"
     option_main_path = 1
     option_side_path = 2
     option_main_world = 3
+    option_beta_biome = 5
```
Comment (alwaysintreble) [inline worlds/noita/Options.py:16]: what's wrong with 4? :think:

Comment (ScipioWright) [reply worlds/noita/Options.py:16]: I'm slowly working on another update that's using the 4 slot, just don't want to steal the number at the moment.

**Comment by heinermann** [comment]
I'm against adding beta content even as an option. We can wait for this to be released to the actual game.

**Comment by ThePhar** [comment]
This has a merge conflict.
image

Also, where do we sit on this regarding this concern?
image

**Comment by ScipioWright** [comment]
I'll just close it for now and pull it again when the beta is finally done.

**Pull request closed without merging**